portfolio.html

```diff
@@ -52,31 +52,57 @@
 							
 				</section>
 		
-				<section class="projects"><div class="contentHeadings"><i class="fas fa-rocket fa-1.2x" style="color:#B4535A"></i> Projects</div>
+				<section class="projects"><div class="contentHeadings"><i class="fas fa-rocket fa-1.2x" style="color:#CE3B45"></i> Projects</div>
 				
-						<div class="experienceItemHeading"> Portfolio Project </div>
-						<div class="experienceItemBody"> I did a week's studing and Mayden accademy and all I got was this dumb profile page</div>
 				
+						<div class="portfolioBanner">
+							<div>
```

Comment:
remove this div layer, dont need it

Reply:
tried to remove, messed up the arrangment a lot though did find one div tag missing a closing slash so maybe that's what threw you